zbus_macros/src/lib.rs

```diff
@@ -215,6 +215,18 @@ pub fn dbus_proxy(attr: TokenStream, item: TokenStream) -> TokenStream {
 /// properties or signal depending on the item attributes. It will implement the [`Interface`] trait
 /// `for T` on your behalf, to handle the message dispatching and introspection support.
 ///
+/// The trait accepts the `interface` attributes:
+///
+/// * `name` - the D-Bus interface name
+///
+/// * `spawn` - By default, zbus will spawn a task for each method call. This may result in methods
+///   being handled out of order and may not be the expected or desired behaviour.
+///
+///   Be careful if you set this attribute to `false`: the object server will block until the
+///   functions returns. In this context, no other methods are handled and the message queues can
+///   become full, blocking the whole connection. You must make sure the implementation doesn't
+///   create such a dead-lock situation, by not blocking/awaiting if possible.
```

Comment:
Well, I don't like vague detailslike "in certain situations" either. but fine.

Reply:
It's less about what I or you like, it's more about what's useful to the user. They don't need to know what internals are involved, just what they have to do, to avoid deadlocks.